### PR TITLE
ENG-9546 Add stream views to MV benchmark

### PR DESCRIPTION
**Pull request by pdshaw**

### CODE REVIEW

File: tests/test_apps/matviewbenchmark/run.sh [hunk 1]
```diff
@@ -60,11 +67,22 @@ function catalog() {
...
+    $VOLTDB compile --classpath obj -o $APPNAME-streamview.jar ddl-streamview.sql
+
     # stop if compilation fails
     if [ $? != 0 ]; then exit; fi
 }
 
 # run the voltdb server locally
+function streamview() {
+    export STREAMVIEW="true"
+    server "-streamview"
```
Comment (prosegay) [inline tests/test_apps/matviewbenchmark/run.sh]: why not just modify APPNAME here ???

**Comment by prosegay** [comment]
you should check charts.dat and see if any of the new charts you're creating will need title settings.

File: tests/test_apps/matviewbenchmark/run.sh [hunk 2]
```diff
@@ -42,10 +45,14 @@ function clean() {
...
+function jars() {
+    ant all
+}
+
 # compile the source code for procedures and the client
 function srccompile() {
     mkdir -p obj
-    javac -classpath $APPCLASSPATH -d obj \
+    javac -source 1.8 -target 1.8 -source 1.8 -target 1.8 -classpath $APPCLASSPATH -d obj \
         src/matviewbenchmark/*.java
```
Comment (prosegay) [inline tests/test_apps/matviewbenchmark/run.sh]: I don't think you wanted to include this change 

git add -p can be your new bff

Comment (pdshaw) [inline tests/test_apps/matviewbenchmark/run.sh]: Removed.

On Fri, Mar 4, 2016 at 4:20 PM, Phil Rosegay <notifications@github.com>
wrote:

> In tests/test_apps/matviewbenchmark/run.sh
> <https://github.com/VoltDB/voltdb/pull/3353#discussion_r55090336>:
>
> >  # compile the source code for procedures and the client
> >  function srccompile() {
> >      mkdir -p obj
> > -    javac -classpath $APPCLASSPATH -d obj \
> > +    javac -source 1.8 -target 1.8 -source 1.8 -target 1.8 -classpath $APPCLASSPATH -d obj \
> >          src/matviewbenchmark/*.java
>
> I don't think you wanted to include this change
>
> git add -p can be your new bff
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/VoltDB/voltdb/pull/3353/files#r55090336>.
>



-- 
*Peter Shaw,* VoltDB Engineering
*VoltDB, Inc.*
*E:* pshaw@voltdb.com

www.voltdb.com <http://voltdb.com>
Follow VoltDB on Twitter <https://twitter.com/voltdb>, Facebook
<https://www.facebook.com/VoltDB?sk=wall>, and LinkedIn
<http://www.linkedin.com/company/voltdb>

**Pull request merged**